tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -105,6 +123,8 @@ class TestCluster(tests.test_resources.test_resource.TestResource):
         ]
     }
 
+    GPU_CLUSTER_NAMES = ["rh-v100", "rh-k80", "rh-a10x"]
```

Comment:
can we extract this more dynamically from the fixtures that use these instance types?